doc/content/xapi/internals/certificates.md

```diff
@@ -97,9 +101,11 @@ used by clients when using HTTPS for API calls.
 * in a plain pool installation, these are empty
 * bundle of public keys provided by a user
 * constructed from PEM files in `certs/`
-* `/opt/xensource/bin/update-ca-bundle.sh` generates bundle from PEM files
+* `/opt/xensource/bin/update-ca-bundle.sh` generates the bundle from PEM files
 * Updated by a user using `xe pool-install-ca-certificate`
 * `Pool.install_ca_certificate`
 * `Pool.uninstall_ca_certificate`
 * `xe pool-certificate-sync` explicitly distribute these certificates in
   the pool.
+* User-provided certificates can be used to let outside services (like
+  WLB) connect to Xapi.
```

Comment:
It's the other way round, xapi uses these certificates in the trust root to connect to external servers, currenly only WLB:
```suggestion
* User-provided certificates can be used to let xapi connect to WLB.
```